src/types/Error.ts

```diff
@@ -0,0 +1,4 @@
+export enum Error {
+  loading = 'Unable to load a todos',
+  add = 'Unable to add a todo',
```

Comment:
first letter of enum keys has to be uppercase
```suggestion
  Loading = 'Unable to load a todos',
  Add = 'Unable to add a todo',
```